### PR TITLE
Feat/disperse siblings after reschedule

### PR DESCRIPTION
**Pull request by L-M-Sherlock**
Patch: [fsrs4anki-helper.ankiaddon.zip](https://github.com/open-spaced-repetition/fsrs4anki-helper/files/15050989/fsrs4anki-helper.ankiaddon.zip)

### CODE REVIEW

**Comment by user1823** [comment]
Doesn't work

Test deck: [Test.apkg.zip](https://github.com/open-spaced-repetition/fsrs4anki-helper/files/15051079/Test.apkg.zip) (Rename to remove .zip)

- Before rescheduling, 19 cards are due.
- After rescheduling (with disperse), 126 cards are due.
- After clicking disperse all siblings, 54 cards are due.

**Comment by L-M-Sherlock** [comment]
New patch: [fsrs4anki-helper.ankiaddon.zip](https://github.com/open-spaced-repetition/fsrs4anki-helper/files/15052948/fsrs4anki-helper.ankiaddon.zip)

File: __init__.py [hunk 1]
```diff
@@ -89,14 +89,24 @@ def set_auto_disperse_after_sync(checked, _):
...
     title="Auto disperse siblings when review", on_click=set_auto_disperse_when_review
 )
 
 
+def set_auto_disperse_after_reschedule(checked, _):
+    config.auto_disperse_after_reschedule = checked
+
+
+menu_auto_disperse_after_reschedule = checkable(
+    title="Disperse sibling after rescheduling (it breaks Load Balance)",
```
Comment (user1823) [inline __init__.py:105]:
```suggestion
    title="Disperse siblings after rescheduling (breaks Load Balance)",
```

File: config.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
     "days_to_reschedule": 7,
     "auto_reschedule_after_sync": false,
     "auto_disperse_after_sync": false,
-    "auto_disperse": false,
+    "auto_disperse_when_review": false,
+    "auto_disperse_after_reschedule": true,
```
Comment (user1823) [inline config.json:9]: It breaks load balance. So, it shouldn't be enabled by default.

```suggestion
    "auto_disperse_after_reschedule": false,
```

**Review by user1823 — COMMENTED** [review]
I won't have the time to test it till tomorrow evening. Till then, I have the following two suggestions.

**Comment by user1823** [comment]
LGTM

**Pull request merged**